sdks/go/pkg/beam/runners/prism/internal/worker/worker.go

```diff
@@ -233,19 +233,19 @@ func (wk *W) GetProcessBundleDescriptor(ctx context.Context, req *fnpb.GetProces
 //
 // Requests come from the runner, and are sent to the client in the SDK.
 func (wk *W) Control(ctrl fnpb.BeamFnControl_ControlServer) error {
-	done := make(chan bool)
+	done := make(chan struct{})
```

Comment:
Question out of curiosity: why replace the boolean with an empty struct?

Reply:
The empty struct is 0 size, and we don't have any semantics around what the channel means. Technically, I should just be closing the channel at this point instead of sending an empty struct but I suspect additional work is going to happen in this area, so I won't change it again here. Currently, not closing the channel doesn't affect whether it's garbage collected after the RPC returns.